fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -49,26 +66,25 @@ public void afterJob() {
             }
         }
 
-        jobEndTimeInMS = System.currentTimeMillis();
-        double jobProcessingSeconds = (jobEndTimeInMS - jobStartTimeInMS)/1000.0;
+
+        double jobProcessingSeconds = (totalJobExecutionMilliSeconds)/1000.0;
```

Comment:
double?  does this need to be a double or is a long just fine?

Reply:
made it double just because the rate is calculated as totalImportedFhirResources/**jobProcessingSeconds**

Reply:
got it... in case, it falls below a second